### PR TITLE
Feature/generate static map

### PR DESCRIPTION
**Pull request by jt8**
Adds getStaticMap() method for generating static maps via the Bing Maps REST API

### CODE REVIEW

File: src/windows/mapProxy.js [hunk 1]
```diff
@@ -167,11 +171,153 @@
...
+        }
+
+        url += (opts.declutterPins) ? "&declutterPins=1" : "";
+        url += (opts.mapSize) ? "&mapSize=" + opts.mapSize.toString() : "";
+
+        return url;
+    }
+
+    map.getStaticMap = function (onSuccess, onError, options) {
+        var opts = {};
```
Comment (simonmarklar) [inline src/windows/mapProxy.js]: Just curious, do we need to do all the validation of options and then rebuilding the object? 

I am assuming bing does all these checks itself and returns an error, so why not just pass through the options value, that way if bing changes the options (eg adds a new option/removes an old one) we don't have to change this ?

**Comment by jt8** [comment]
Yeah, true. I'll make the changes

**Comment by jt8** [comment]
I've kept the validation for the bing maps key, because you can set it via setOptions() before calling this method.

**Comment by simonmarklar** [comment]
:+1: lgtm

**Comment by jokeyrhyme** [comment]
I like that you're passing an options Object, that'll leave room for maybe asking for other return types later (e.g. Cordova's FILE_URL vs DATA_URI, etc).
And the documentation additions are good, too.

LGTM :+1:

**Pull request merged**